src/formatDate.js

```diff
@@ -8,7 +8,65 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const YEAR_SHORT_FORMAT = 'YY';
+  const YEAR_LONG_FORMAT = 'YYYY';
+  const MONTH_FORMAT = 'MM';
+  const DAY_FORMAT = 'DD';
+  const CENTURY_21 = '20';
+  const CENTURY_20 = '19';
+  const MAX_YEAR = 30;
+  const DELIMITER = fromFormat.pop();
+  const SEPARATOR = toFormat.pop();
```

Comment:
Modifying the input parameters directly by using pop() can lead to unexpected side effects if the fromFormat and toFormat arrays are used elsewhere after calling this function. It's better to create a new variable for the delimiters that doesn't alter the original arrays.